examples/scripts/online_dpo.py

```diff
@@ -96,8 +96,9 @@ def tokenize(element):
     if tokenizer.chat_template is None:
         tokenizer.chat_template = SIMPLE_QUERY_CHAT_TEMPLATE
     reward_model = AutoModelForSequenceClassification.from_pretrained(config.reward_model_path, num_labels=1)
-    ref_policy = AutoModelForCausalLM.from_pretrained(config.sft_model_path)
-    policy = AutoModelForCausalLM.from_pretrained(config.sft_model_path)
+    ref_model = AutoModelForCausalLM.from_pretrained(config.sft_model_path)
+    model = AutoModelForCausalLM.from_pretrained(model_config.model_name_or_path)
```

Comment:
@edbeeching i am using the `model_name_or_path` for the main model for which the `tokenizer` was initialized (rather than the `sft_model_path` which i believe was for the `ref_model`)